Eastern-Research-Group/mywaterway
# Feature/137 update epa template

## Related Issues:
* [HMW-137](https://jira.epa.gov/browse/HMW-137)

## Main Changes:
* Updated the base HMTL template to the current one stipulated by EPA.
* Included all necessary EPA files locally.
* Addressed styles that were broken by the update.

## Steps To Test:
1. Test as many pages as possible and verify the appearance of all non-template components are the same as before.
2. Ensure the template is updated on the `400.html` and `500.html` pages.

## NOTES:
All of the auxiliary template files are located under `public/epa-template-files` in both the server and client, so that they could be easily copied/synced. The duplicates will be consolidated when building the application, but there could be a better way to make the files reachable by React during development (I considered symlinks, but I don't think they would play well between operating systems).
